### DISCUSSION TITLE
OpenWebUI Daniel Miessler Fabric Patterns !

### DISCUSSION THREAD

**Discussion by freeload101** [open-webui/open-webui#7737] [Show and tell]
https://youtu.be/lmqFpNq-gPg

**Comment by etlweather** [comment]
Do you have them so they can be all imported automatically?

**Reply by freeload101** [reply]
I mean you can probably use API to do that. Submit a issue to my SCRIPTS main repo I think it's in 
?
http://host.docker.internal:3000/api/v1/prompts/command/test/update
